fala/templates/generic-py-base.html

```diff
@@ -8,7 +8,7 @@
     {% block stylesheets %}{% endblock %}
     {% block head %}{% endblock %}
   </head>
-  <body class="govuk-template__body ">
+  <body class="govuk-template__body {{ 'cla-' + ENVIRONMENT }}">
```

Comment:
I don't think we should use the prefix `cla-`, perhaps `fala-` is more suited.  The CSS will need changing too.